chore: README improvements (#2987)
paperless-ngx/paperless-ngx: README.md

## Bugs

> For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/issues), or [start a discussion](https://github.com/paperless-ngx/paperless-ngx/discussions) in case you have questions.

Reviewer: ```suggestion
For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/issues) or [start a discussion](https://github.com/paperless-ngx/paperless-ngx/discussions) if you have questions.
```

# Affiliated Projects

> Paperless has been around a while now, and people started to build tools that interact with it. If you're one of those people, we can add your project to this list:

Reviewer: ```suggestion
Paperless has been around for a while now, and people have built tools that interact with it. If you're one of those people, please reach out and we can add your project to the list. Current projects include:
```

- **Mobile**
  - [Paperless App](https://github.com/bauerj/paperless_app): An Android/iOS application for Paperless-ngx.
  - [Paperless Mobile](https://github.com/astubenbord/paperless-mobile): A modern, feature rich Android app for Paperless-ngx.
  - [Paperless Share](https://github.com/qcasey/paperless_share): Share any files from your Android application with Paperless-ngx. Very simple, but works with all mobile scanning apps that allow you to share scanned documents.
- **Desktop**
  - [Scan to Paperless](https://github.com/sbrunner/scan-to-paperless): Scan and prepare (crop, deskew, OCR, ...) your documents for use in Paperless-ngx.

# Important Note

> > Document scanners are typically used to scan sensitive documents. Things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. Instead, we recommend that if you do want to use it, **run it locally on a server in your own home**.

Reviewer: ```suggestion
> Document scanners are typically used to scan sensitive documents—things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. No guarantees are made regarding security (but we do try!) and **the safest way to run Paperless-ngx is on a local server eg in your own home**. In other words, **use at your own risk**.
```

# Affiliated Projects

Paperless has been around for a while now, and people have built tools that interact with it. If you're one of those people, please reach out and we can add your project to the list. Current projects include:

- **Mobile**
  - [Paperless App](https://github.com/bauerj/paperless_app): An Android/iOS application for Paperless-ngx.
  - [Paperless Mobile](https://github.com/astubenbord/paperless-mobile): A modern, feature rich Android app for Paperless-ngx.
  - [Paperless Share](https://github.com/qcasey/paperless_share): Share any files from your Android application with Paperless-ngx. Very simple, but works with all mobile scanning apps that allow you to share scanned documents.
- **Desktop**
  - [Scan to Paperless](https://github.com/sbrunner/scan-to-paperless): Scan and prepare (crop, deskew, OCR, ...) your documents for use in Paperless-ngx.

# Important Note

> > Document scanners are typically used to scan sensitive documents—things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. No guarantees are made regarding security (but we do try!) and **the safest way to run Paperless-ngx is on a local server eg in your own home**. In other words, **use at your own risk**.

Reviewer: ```suggestion
> Document scanners are typically used to scan sensitive documents, things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. No guarantees are made regarding security (but we do try!) and **the safest way to run Paperless-ngx is on a local server eg in your own home**. In other words, **use at your own risk**.
```
